src/components/defaultLayout/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Account from '../account';
+import PrivateRoute from '../privateRoute';
+
+const DefaultLayout = ({ component: Component, ...rest }) => (
+  <PrivateRoute {...rest} component={ matchProps => (
+    <main>
+      <Account />
```

Comment:
I noticed you render Account component and Link components every time changing the route,  I think rending them once will help improving performance. to do so, I’d suggest find a way to keep those components mounted once or adapting unit tests. nested routes are used in many apps. there should be some other applications facing this situation.